packages/terra-core-docs/src/terra-dev-site/doc/form-select/example/combobox/Invalid.jsx

```diff
@@ -1,18 +1,48 @@
-import React from 'react';
+import React, { useState } from 'react';
 import Combobox from 'terra-form-select/lib/Combobox';
 import classNames from 'classnames/bind';
+import VisuallyHiddenText from 'terra-visually-hidden-text';
 import styles from '../FormSelectDocCommon.module.scss';
 
 const cx = classNames.bind(styles);
 
-const InvalidExample = () => (
-  <Combobox placeholder="Select a color" isInvalid className={cx('form-select')}>
-    <Combobox.Option value="blue" display="Blue" />
-    <Combobox.Option value="green" display="Green" />
-    <Combobox.Option value="purple" display="Purple" />
-    <Combobox.Option value="red" display="Red" />
-    <Combobox.Option value="violet" display="Violet" />
-  </Combobox>
-);
+const InvalidExample = () => {
+  const [isInvalid, setIsInvalid] = useState(true);
+  const [selectedValue, setSelectedValue] = useState('');
+  const [invalidText, setInvalidText] = useState(null);
 
+  const handleSelectChange = (value) => {
+    setIsInvalid(false);
+    setSelectedValue(value);
+  };
+
+  const handleFocus = () => {
+    setInvalidText('Please select a color');
```

Comment:
this should be a initial value of InvalidText state and should get cleared out on selection of option

Reply:
Hi @supreethmr ,
if we set this text as initial value SR Identifies the visually hidden text and reads it on the page load. so we are doing it on when select receives focus. And "should get cleared out on selection of option". This changes are done.
Thanks

Reply:
I would suggest to keep the code consistent with search-invalid example !!

Reply:
We have removed aria-invalid changed in this PR